src/python_testing/TC_IDM_10_7.py

```
@@ -0,0 +1,359 @@
+        asserts.assert_is_not_none(commissioning_method, "in_test_commissioning_method must not be None")
+        asserts.assert_true(
+            str(commissioning_method).startswith("nfc-"),
+            f"Expected in_test_commissioning_method to start with 'nfc-', got: {commissioning_method}"
+        )
+
+        self.step(1)    # Perform a 'PASE only' session, over NTL
+
+        log.info("dut_node_ids 0x%X", node_id)
+
+        nfc_reader_index = self.user_params.get("NFC_Reader_index", 0)
+        reader = matter.testing.nfc.NFCReader(nfc_reader_index)
```

Flake8 (7.3.0)

[error] undefined name 'matter'

(F821)